speakeasy/binemu.py

```diff
@@ -795,6 +795,9 @@ def write_mem_string(self, string, address, width=1):
         else:
             raise ValueError('Invalid string encoding')
 
+        if not string.endswith("\0"):
+            string += "\0"
```

Comment:
i think many strings in this project are single quoted, but here they are double quoted.

just a nit.

Reply:
changed, good catch